eunyoung/app.js

```diff
@@ -71,22 +72,109 @@ app.post("/posts", async (req, res, next) => {
     res.status(201).json({ "message" : "postCreated!" });
 })
 
-app.get("/posts", async (req, res, next) => {
+// 전체 게시글 조회
+app.get("/posts/all", async (req, res, next) => {
     const postData = 
     await myDataSource.query(
         `SELECT
+            users.id AS userID,
+            users.userImg as userProfileImage,
+            posts.userId AS postingID,
+            posts.contentImg AS postingImageUrl,
+            posts.content AS postingContent
+        FROM
+            users
+        INNER JOIN
+            posts ON posts.userId = users.id   
+        `
+    );
+    res.status(200).json({ data: postData });
+})
+
+// 유저게시글 조회
+app.get("/posts/:userId", async (req, res) => {
```

Comment:
```suggestion
app.get("/posts/user/:userId", async (req, res) => {
```
작성하신대로면 `:3000 /posts/1`처럼 보이게 됩니다. 그보다는 제가 위와 같이 수정하는 것이 자원을 명확하게 표현한 것이라고 볼 수 있습니다~